sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/src/main/java/com/microsoft/azure/servicebus/stream/binder/ServiceBusMessageChannelBinder.java

```diff
@@ -78,6 +84,11 @@ public String getDefaultsPrefix() {
         return this.bindingProperties.getExtendedPropertiesEntryClass();
     }
 
+    @Override
+    protected ErrorMessageStrategy getErrorMessageStrategy() {
+        return DEFAULT_ERROR_MESSAGE_STRATEGY;
```

Comment:
@chenghaoharvey Thanks for your great effort. I just have one concern that need the `ErrorMessageStrategy` be hard coded?

Reply:
i just refer to rabbit mq implementation. 

you mean that message strategy should not be hard coded and user can config it by themselves?
Or i just `return new ErrorMessageStrategy()` directly?

Reply:
@yiliuTo may i have your input? I referred to Rabbit MQ binder and they are hard code as well.

Reply:
hihi，any response?

Reply:
I am so sorry that I have ignored this notification for so many days==. After investigating code of `ErrorMessageStrategy` from both Rabbit MQ binder and azure0-spring-integration-core project, I think it makes sense to leave it as hard coded.